ultraviolet/shared/src/main/scala/ultraviolet/macros/CreateShaderAST.scala

```diff
@@ -1694,5 +1764,5 @@ class CreateShaderAST[Q <: Quotes](using val qq: Q) extends ShaderMacroUtils:
         ShaderAST.While(walkTerm(cond, envVarName), walkTerm(body, envVarName))
 
       case x =>
-        val sample = Printer.TreeStructure.show(x).take(100)
+        val sample = Printer.TreeStructure.show(x).take(1000)
```

Comment:
Might want to knock this back down again before we merge.

Reply:
Reverted https://github.com/PurpleKingdomGames/ultraviolet/pull/142/commits/982b1e8062d5ce9e9b664b3487e84de2de8aae10